code/modules/mob/living/living.dm

```diff
@@ -402,6 +402,7 @@
 	if(src)
 		SEND_SIGNAL(src, COMSIG_MOB_MOVE_OR_LOOK, TRUE, dir, dir)
 	if(!istype(LM) || !LM.target || !src || buckled)
+		LM.invoke_end_throw_callbacks(src)
```

Comment:
Are you certain all the conditions that were returning for here allow us to be able to even call the end throw callbacks? E.g. if the istype fails, will it even be something that implements this proc? And are there any callbacks added to the end throw callbacks that need src (move_atom) or the target to not be falsey?

You probably only want the buckled return, not the others.

Reply:
I admit I didn't even think of that I saw buckled and just whacked it in there.